spec/requests/campaigns_request_spec.rb

```diff
@@ -171,6 +181,35 @@
             expect(campaign.valid).to eq true
           end
         end
+
+        context 'with valid parameters and project_id' do
+          let(:location_id) { @location.id }
+          let(:seller_id) { nil }
+          let(:project_id) { @project.id }
+          let(:distributor_id) { distributor.id }
+
+          it 'returns status code 201' do
+            expect(response).to have_http_status(201)
+          end
+
+          it 'creates a Campaign with default values and matching attributes' do
+            response_body = JSON.parse(response.body)
+            expect(response_body).not_to be_nil
+            expect(json['amount_raised']).to eq 0
+            expect(json['last_contribution']).to eq nil
+
+            campaign = Campaign.find(response_body['id'])
+            expect(campaign).not_to be_nil
+            expect(campaign.location).to eq @location
+            expect(campaign.project).to eq @project
+            expect(campaign.target_amount).to eq 100_000
+            expect(campaign.amount_raised).to eq 0
+            expect(campaign.price_per_meal).to eq 500
+
+            expect(campaign.active).to eq false
```

Comment:
I think to be_true/be_false is a thing

Reply:
changed to 'expect( ).to be true' since I think be_true checks for truthiness vs the actual value